client/me/billing-history/style.scss

```diff
@@ -352,6 +358,10 @@ textarea.billing-history__billing-details-editable {
 	padding: 0;
 }
 
+.billing-history {
+	min-height: 500px;
```

Comment:
As far as I can tell, we probably don't need this. Can you share why you added it?

Reply:
Without it, the drop-down menu for the date was cut off at the bottom when there were no purchases listed. That _was_ before I added the bottom navigation link, so maybe now it gives us enough pixels to work with... but I'd still feel better if this was here to prevent cutting off the menu (which always contains 6 months and a few other options).

Reply:
![Screen Shot 2020-10-06 at 7 21 10 PM](https://user-images.githubusercontent.com/2036909/95270272-2eeddb00-0809-11eb-9e9e-ea0f2b8bfd01.png)

Reply:
@fditrapani If you have a better solution I'm happy to use it!

Reply:
aah ok — that makes sense. Nothing immediately comes to mind. It likely has more to do with the dropdown than it does with the container. Probably nothing worth worrying about right now. Thanks!